### PR TITLE
fix(bazel): fix integration test for bazel building

### PR DESCRIPTION
**Pull request by josephperrott**
Update the API used to request a timestamp.

### CODE REVIEW

**Review by atscott — APPROVED** [review]
LGTM since the test now passes but can you update the commit message to be a bit more descriptive about _why_ this change was made? It currently just says "what" the change is but not "why" it was necessary.

File: integration/bazel/src/app.component.ts [hunk 1]
```diff
@@ -15,6 +15,6 @@ export class AppComponent {
   constructor(private http: HttpClient) {}
 
   time$: Observable<string> =
-      this.http.get('http://worldclockapi.com/api/json/pst/now')
-          .pipe(map((result: any) => result.currentDateTime), startWith(['...']));
+      this.http.get('http://worldtimeapi.org/api/timezone/America/Los_Angeles.json')
```
Comment (devversion) [inline integration/bazel/src/app.component.ts:18]: Out of curiosity: Should we just change this away from an external server?

Comment (josephperrott) [reply integration/bazel/src/app.component.ts:18]: Yes, my plan was to investigate further to determine if we need this at all, but wanted to get CI fixed first.

Comment (devversion) [reply integration/bazel/src/app.component.ts:18]: I see, nice catch by the way 😄!

**Pull request closed without merging**